tcms/testruns/tests/test_admin.py

```diff
@@ -47,3 +48,23 @@ def test_superuser_can_delete_testrun(self):
                                             args=[self.test_run.pk]), {'post': 'yes'}, follow=True)
         self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
         self.assertEqual(TestRun.objects.filter(pk=self.test_run.pk).exists(), False)
+
+
+class TestTestExecutionStatusAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.superuser = UserFactory()
+        cls.superuser.is_superuser = True
+        cls.superuser.set_password('password')
+        cls.superuser.save()
+
+    def test_admin_testrun_execution_status_color(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.superuser.username,
+            password='password')
+        response = self.client.get(reverse('admin:testruns_testexecutionstatus_changelist'))
+        self.assertContains(response, '#92d400')
+        self.assertContains(response, '#72767b')
+        self.assertContains(response, '#cc0000')
```

Comment:
All of these are field values which will be displayed regardless of whether or not the cell displaying them changes it's background color. The important behavior of the `colored_color` method/column is that the background color matches the actual value using CSS attributes.